src/test/run-pass/allocator/custom.rs

```diff
@@ -39,8 +38,7 @@ unsafe impl<'a> Alloc for &'a A {
 static GLOBAL: A = A;
 
 fn main() {
-    env::set_var("FOO", "bar");
-    drop(env::var("FOO"));
+    println!("hello!");
```

Comment:
Stray print?

Reply:
Nah this is just intended to force an allocation somewhere that LLVM can't optimize away